src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+.page {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  margin: 0;
+
+  &__content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
I am not sure, but try leaving one value since they are the same.

Reply:
I need to use both because the calendar moves too far down and to the left